### DISCUSSION TITLE
How to use WoltModalSheet in Widget?

### DISCUSSION THREAD

**Discussion by Trung15010802** [woltapp/wolt_modal_sheet#313] [Q&A] [answered]
I want to use WoltModalSheet  in StatefulWidget but I don't know now. Right now I'm using `show `method to open modal

**Comment by ulusoyca** [comment]
This is being worked nowadays. @MbIXjkee is coordinating the project.

**Reply by MbIXjkee** [reply, marked as answer]
Yes, we are currently working on a huge update.
There should be an opportunity to use WoltModalSheet widget directly, use widgets directly as content for WoltModalSheet, have custom navigation inside, and a lot of additional things that allow direct customization for the users of the library.

Stay tuned!

**Reply by Trung15010802** [reply]
Amazing, I’m really looking forward to it.

**Reply by namantam1** [reply]
Any updates on it?, I am waiting for this feature :)

**Reply by ulusoyca** [reply]
Yes, it is a big change, and we are in touch. Last night, I worked on integrating the solution @MbIXjkee came up with, and today we have meeting.
